src/config/pci-passthrough.md

```diff
@@ -0,0 +1,405 @@
+# PCI Passthrough
+
+PCI pass-through allows you to pass through an extra graphics card in your
+system with little to no performance drawback. This allows you to run
+applications that require GPU acceleration (CAD programs, video games, etc.) in
+your virtual machine on operating systems like Windows 10 with almost bare metal
+performance.
+
+PCI pass-through requires a lot of trial and error. I recommend you read the
+external resources and read each section from beginning to end.
```

Comment:
Not sure either of these sentences are needed, but if so, the second sentence should be rewritten to not use first-person, e.g. something along the lines of: "Be sure to read this section carefully, as well as the external resources provided."

Reply:
Fixed